core/sql/executor/ExUdrServer.cpp

```diff
@@ -298,7 +287,7 @@ ExUdrServer::ExUdrServerStatus ExUdrServer::start(ComDiagsArea **diags,
     ipcServer_ =
       udrServerClass_->allocateServerProcess(diags,
                                              diagsHeap,
-                                             nodeName,
+                                             NULL,
```

Comment:
Would the udr server still be colocated with the ESP ? It needs to know the nodename ot start it right ?

Reply:
Yes. This is determined by the next parameter, "collocatedCPU".

Reply:
Ok